CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 ## 1.6.4 *Aug 30th 2022*
 Fixed:
 
+ * Add hex and octal BitVector parsing. [#1772] (https://github.com/clash-lang/clash-compiler/pull/2505)
```

Comment:
Change files should be added to `changelog/`. See `changelog/README.md` for more details :).

Reply:
Moved! Hopefully I did it right.